ccm_web/client/src/components/GradebookCanvasValidators.ts

```diff
@@ -1,11 +1,8 @@
 import { InvalidationType } from '../models/models'
```

Comment:
Optional: I saw a couple of files as Validators, could go inside a sub-folder in component as `validator` for code organization purpose.

Reply:
That would be great, I'll consider doing that in the future, but I didn't want to change even more in this PR.